test/test_nn.py

```diff
@@ -3353,12 +3353,27 @@ def test_embedding_from_pretrained(self):
         output = embedding(input)
         self.assertEqual(a, output)
 
+    def test_embedding_bag_from_pretrained(self):
+        a = torch.Tensor([[1, 2, 3], [4, 5, 6]])
+        embedding = nn.EmbeddingBag.from_pretrained(a)
+        self.assertEqual(a, embedding.weight.data)
```

Comment:
`.data` is old syntax and no longer needed. Please update throughout the newly added tests.